src/otg/assets/schemas/study_locus.json

```diff
@@ -82,6 +88,18 @@
       "nullable": true,
       "type": "string"
     },
+    {
+      "metadata": {},
+      "name": "credibleSetIndex",
+      "nullable": true,
+      "type": "integer"
+    },
+    {
+      "metadata": {},
+      "name": "credibleSetlog10BF",
```

Comment:
Do we need to keep this field? LogBF is part of `locus` already

Reply:
same question

Reply:
@addramir mentioned in a meeting today that the BF here is relative to the credible set as a whole, so it makes sense to add it at the association level.